src/rtctools_heat_network/workflows/io/write_output.py

```diff
@@ -965,8 +978,8 @@ def _name_to_asset(name):
                         profiles.profile_data_list.append(data_row)
                     # end time steps
                     profiles.num_profile_items = len(profiles.profile_data_list)
-                    profiles.start_datetime = pytz.utc.localize(profiles.profile_data_list[0][0])
-                    profiles.end_datetime = pytz.utc.localize(profiles.profile_data_list[-1][0])
+                    profiles.start_datetime = profiles.profile_data_list[0][0]
```

Comment:
Originally you localized this datetime to utc. It is okay if it is any timezone?

Reply:
Thanks for the info Kobus! :D